news.html

```diff
@@ -98,13 +115,18 @@ <h2><a href="#">Herman Cain, US ex-presidential candidate, dies after contractin
         </article>
     </section> <br><br>
     <section>
-        <h2><a href="#">NIGERIA, IMF AND COVID-19</a></h2>
-        <b>Source:</b> CISLAC Nigeria <br>
-  		<b>Date:</b> 16th July, 2020 <br>
+        <a id="headline4">
+        <h2>NIGERIA, IMF AND COVID-19</h2>
+        <a id="fourth-image">
```

Comment:
You didn't wrap the image with the a tag here...

Reply:
> You didn't place the name in the image tag here?
> I think the ID is supposed to be in the tag you are pointing to and not alone. Am i wrong?

That is how it is supposed to be. You are just supposed to place it anywhere on the webpage.